src/main/java/cz/cvut/kbss/termit/service/repository/ResourceRepositoryService.java

```diff
@@ -70,6 +74,30 @@ public ResourceRepositoryService(Validator validator, ResourceDao resourceDao,
         return resourceDao;
     }
 
+    @Cacheable
+    @Override
+    public List<Resource> findAll() {
+        return super.findAll();
+    }
+
+    @CacheEvict(allEntries = true)
```

Comment:
why these methods evict all entries?

Reply:
See javadoc to `CacheConfig` - there is actually only one entry in the cache and it contains the whole list of elements.

Reply:
ok, makes sense.